tests/test-spawn-snid.lua

```diff
@@ -0,0 +1,44 @@
+--- Tests for spawn's startup notifications.
+
+local spawn = require("awful.spawn")
+
+local manage_called, c_snid
+
+client.connect_signal("manage", function(c)
+  manage_called = true
+  c_snid = c.startup_id
+  print("manage_called: c_snid:", tostring(c_snid))
```

Comment:
This PR is based on the old Travis infrastructure (Ubuntu Precise), where it seems to be just missing: http://packages.ubuntu.com/precise/rxvt-unicode